DEVELOP.md

```diff
@@ -11,7 +11,7 @@ npm version patch  # or minor or major
 npm publish
 ```
 
-* To run or deploy the simple test app on Cloudflare, create a `.dev.vars` file containing `DATABASE_URL=postgres://connection_string`, run `npx wrangler dev --local` or `npx wrangler publish`.
+* To run or deploy the simple test app on Cloudflare, create a `.dev.vars` file containing `NEON_DB_URL=postgres://connection_string`, run `npx wrangler dev --local` or `npx wrangler publish`.
```

Comment:
why this change? Is it intentional?

Reply:
Yes, the previous description was incorrect.

Reply:
I assume, users can use any format they want, but ok

Reply:
Unfortunately, no, I struggled a lot with using `DATABASE_URL` and it didn't work.

Reply:
oh wow, sorry, I was under impression that this is just an env to var to user in your backend

Reply:
Sorry @khanova, you were right to switch this. It _is_ just an arbitrary env var name, but it's one that's expected by the code in `src/index.ts`, for testing on Cloudflare.